.github/workflows/release.yml

```diff
@@ -101,14 +97,39 @@ jobs:
       - name: "Read changelog from draft release"
         run: gh release view ${{ env.PACKAGE_VERSION }} --json body --template '{{ .body }}' >> changelog
 
-      # TODO: Sign tag
-      - name: "Create release tag"
-        run: git tag -a -F changelog ${{ env.PACKAGE_VERSION }}
-
-      - name: "Update version information to next patch development release"
+      - name: "Prepare tag message"
         run: |
-          ./bin/update-release-version.php to-next-patch-dev
-          git commit -m "Back to -dev" phongo_version.h
+          echo "Release ${PACKAGE_VERSION}" > tag-message
```

Comment:
Can we add an additional line break after this to segregate the first line of the tag message from its body (as we do for commits)?

Reply:
Added. In my previous attempts, this printed a literal `\n`, which I now realised can be fixed by using `-e` with echo.